### DISCUSSION TITLE
Icon is not used on Windows

### DISCUSSION THREAD

**Discussion by nazar-pc** [hoodie/notify-rust#229] [Q&A]
On Windows only `image_path` that is set using `Toast::image()` even though there is `Toast::icon()` method too that could have benefited from an absolute path to an icon on Windows.

An image is not the same as icon on Windows and is shown much larger below the text.

UPD: Calling `app_id` after figuring it out (for installed app) using `get-StartApps` in PowerShell sets both icon and app name properly.

**Comment by hoodie** [comment]
thanks for posting the solution!

**Reply by nazar-pc** [reply]
I would argue `image_path` should probably not use `Toast::image()` still, at least it is not consistent with how things look on Linux in that case.
